ReferenceAppKotlin/app/src/main/java/com/android/tv/reference/watchnext/WatchNextHelper.kt

```diff
@@ -143,8 +143,6 @@ object WatchNextHelper {
     /**
      * Retrieve all programs in Watch Next row.
      */
-    @SuppressLint("RestrictedApi")
```

Comment:
Unfortunately the referenced library is restricted, so this lint suppression is needed here.

Reply:
This removal can be merged now that [bug #138150076](https://issuetracker.google.com/138150076) has been resolved.